src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -2136,12 +2136,19 @@ protected function compare($first, $second, $operator)
     public function parseNamedParameters($parameters)
     {
         return array_reduce($parameters, function ($result, $item) {
-            [$key, $value] = array_pad(explode('=', $item, 2), 2, null);
-
-            $result[$key] = $value;
+            // If the item is a an array, we will assume the parameters have been
+            // passed in as groups of key value pairs. Otherwise, we can assume
+            // they are string based, separating them by their equals sign.
+            if (is_array($item)) {
+                array_push($result, $item);
+            } else {
+                [$key, $value] = array_pad(explode('=', $item, 2), 2, null);
+
+                $result[$key] = $value;
+            }
 
             return $result;
-        });
+        }, []);
```

Comment:
This change has been added so that parameters that have been given as `key => value` pairs (such as with the `dimensions` rule), are properly pushed onto the result, rather than attempting to explode the array, throwing an exception.